definitions/variable/technology/electricity-operation.yaml

```diff
@@ -12,360 +12,82 @@
-    description: Operation Cost of New+Imported biomass power plants biomass from
-      dedicated energy crops and biomass imports.
-    unit: EUR_2020
-    skip-region-aggregation: true
-- Operation Cost|Electricity|Nuclear:
-    description: Operation Cost of nuclear power plants
-    unit: EUR_2020
-    skip-region-aggregation: true
-- Operation Cost|Electricity|Geothermal:
-    description: Operation Cost of Geothermal power plants
-    unit: EUR_2020
-    skip-region-aggregation: true
-- Operation Cost|Electricity|Gas:
-    description: Operation Cost of Gas power plants
-    unit: EUR_2020
-    skip-region-aggregation: true
-- Operation Cost|Electricity|Gas|OCGT:
-    description: Operation Cost of Open Cycle Gas Turbine power plants
-    unit: EUR_2020
-    skip-region-aggregation: true
-- Operation Cost|Electricity|Gas|CCGT:
-    description: Operation Cost of Combined Cycle Gas Turbine power plants
-    unit: EUR_2020
-    skip-region-aggregation: true
-- Operation Cost|Electricity|Coal:
-    description: Operation Cost of coal power plants
-    unit: EUR_2020
-    skip-region-aggregation: true
-- Operation Cost|Electricity|Lignite:
-    description: Operation Cost of Lignite power plants
-    unit: EUR_2020
-    skip-region-aggregation: true
-- Operation Cost|Electricity|Oil:
-    description: Operation Cost of Oil power plants
-    unit: EUR_2020
-    skip-region-aggregation: true
-- Operation Cost|Electricity|Hydro|Reservoir:
-    description: Operation Cost of "simple Hydro reservoir " units
-    unit: EUR_2020
-    skip-region-aggregation: true
-- Operation Cost|Electricity|Hydro|Pumped Storage:
-    description: Operation Cost of "Pumped Storage " units
-    unit: EUR_2020
-    skip-region-aggregation: true
-- Operation Cost|Electricity|Hydro|Run of River:
-    description: Operation Cost of "Run of River" units
-    unit: EUR_2020
-    skip-region-aggregation: true
-- Operation Cost|Electricity|Hydro|Ocean:
-    description: Operation Cost of Ocean Power plants
-    unit: EUR_2020
-    skip-region-aggregation: true
-- Operation Cost|Electricity|Solar:
-    description: Operation Cost of Photovoltaic power plants
-    unit: EUR_2020
-    skip-region-aggregation: true
-- Operation Cost|Electricity|Wind|OnShore:
-    description: Operation Cost of  Wind Onshore power plants
+    unit:
+    - MWh
+    - GWh
+
+- Operation Cost|Electricity|<Fuel>:
+    description: Operation Cost of <this fuel> power plants
     unit: EUR_2020
     skip-region-aggregation: true
-- Operation Cost|Electricity|Wind|OffShore:
-    description: Operation Cost of  Wind OffShore power plants
-    unit: EUR_2020
+
+- Reserve|Electricity|Automatic Frequency Restoration|<Fuel>:
+    description: Automatic Frequency Restoration Reserve available for <this fuel>
+     power plants; this amount is the maximum available for the grid operator considering
+     the commitment decisions taken by the model
+    unit:
+    - MWh
+    - GWh
```

Comment:
Is there a particular reason why this variable could not be aggregated by simple summation across regions? ie., why have `skip-region-aggregation: true`

Reply:
It could open another discussion; I would prefer not to aggregate according to our model approach. Still, in general terms, it could be used for users to talk about the reserve at different regional aggregations. Then, I'll change these variables to be aggregated.

Reply:
Got it, ok

Reply:
You convinced me, you please revert to `skip-region-aggregation: true`

Reply:
Ok, I changed it to `skip-region-aggregation: true`.

Reply:
@erikfilias why do you forbid aggregation of reserves? When aggregating 2 regions, the resulting aFR per fuel type is the sum of both, so I wouldn't have prevented this....

Reply:
> It could open another discussion; I would prefer not to aggregate according to our model approach. Still, in general terms, it could be used for users to talk about the reserve at different regional aggregations. Then, I'll change these variables to be aggregated.

I don't deny that it could be aggregated. We are not using the reserves but I also agree to leave the aggregation as an option. In this sense, I will change to `skip-region-aggregation: False` .